package.json

```diff
@@ -42,6 +42,8 @@
     "semantic-release": "^19.0.5",
     "serverless": "^3.17.0",
     "serverless-esbuild": "^1.37.2",
+    "serverless-bundle": "^6.0.0",
```

Comment:
@mdial89f  this came through with the update from base. but i cant tell where its being used in either project. i think maybe it should be used instead of the serverless-plugin-typescript